react/SliderLayout.tsx

```diff
@@ -16,18 +16,25 @@ const SliderLayout: StorefrontFunctionComponent<SliderLayoutProps &
   fullWidth = true,
   arrowSize = 25,
   children,
+  itemsPerPage = {
+    desktop: 5,
+    tablet: 3,
+    phone: 1,
+  },
```

Comment:
This might be a breaking change, no?

Reply:
Nope, I've just changed where this default value is defined, but this was already its value